### PR TITLE
how to convert to namespace project

### PR DESCRIPTION
**Pull request by cchris28**
This documents a manual conversion of a python-project-template to a namespace project.

### CODE REVIEW

**Comment by drewoldag** [comment]
Looks like the version of copier has been updated to 8.0.0. Here's a list of the changes: https://github.com/copier-org/copier/releases/tag/v8.0.0 

It seems like we need to change the command that we're using to generate the projects. 

In particular line 85 of ci.yml looks like it needs to be changed from this: 
`copier --vcs-ref HEAD --defaults ${{ matrix.copier_config.extra_flags }} copy ./ ../test/${{ matrix.copier_config.foldername }}`
to this:
`copier copy ./ ../test/${{ matrix.copier_config.foldername }} --vcs-ref HEAD --defaults ${{ matrix.copier_config.extra_flags }}`

The difference being that the copy subcommand comes before flags. At least that's what I'm getting from the release notes without actually trying it on the command line.

File: docs/practices/namepace_project.rst [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    from acme.utilities import fits, wcs
+
+A python project template created project can be converted to a `pep 420 <https://peps.python.org/pep-0420/>`_ compliant namespace package with a few steps.
+The example provided here is for a new project but the procedure can be applied to existing projects that have not strayed signficiantly from the template file structure.
+
+Synopsis
+--------
+
+#. Choose an organization namespace shared by all related packages. For example **acme**.
+#. Rename the **src** directory to to organization name **acme**.
```
Comment (delucchi-cmu) [inline docs/practices/namepace_project.rst:21]:
```suggestion
#. Rename the **src** directory to the organization name **acme**.
```

File: docs/practices/namepace_project.rst [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+#. Replaces references to **src** with references to **acme**.
```
Comment (delucchi-cmu) [inline docs/practices/namepace_project.rst:22]:
```suggestion
#. Replace references to **src** with references to **acme**.
```

File: docs/practices/namepace_project.rst [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    'Hello from LINCC-Frameworks!'
+    >>> utilities.greetings()
+    'Hello from LINCC-Frameworks!'
+    >>> utilities.meaning()
+    42
+    >>> exit()
+
+You might want to uninstall the packages to avoid python environment bloat!
+
+.. code-block: bash
```
Comment (delucchi-cmu) [inline docs/practices/namepace_project.rst:162]:
```suggestion
.. code-block:: bash
```

**Review by delucchi-cmu — APPROVED** [review]

**Pull request merged**